lib/pageflow/file_types.rb

```diff
@@ -7,13 +7,16 @@ def initialize(page_types)
     end
 
     def each(&block)
-      @page_types.map(&:file_types).flatten.uniq(&:model).each(&block)
+      first_level_file_types = @page_types.map(&:file_types).flatten
+      lower_level_file_types = search_for_nested_file_types(first_level_file_types)
+      (first_level_file_types + lower_level_file_types).uniq(&:model).each(&block)
     end
```

Comment:
I think in this case we already do what you asked for here: https://github.com/codevise/pageflow/pull/660/files#diff-f26ff2c97b6893fedefc1ce6225aabd3L17